service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -59,129 +57,75 @@ void shouldThrowExceptionWhenMigrationNotMappedForMigrationID() {
             .hasMessage("No migration mapped to " + INVALID_MIGRATION_ID);
     }
 
-    private CaseDetails buildCaseDetails(CaseData caseData, String migrationId) {
-        CaseDetails caseDetails = asCaseDetails(caseData);
-        caseDetails.getData().put("migrationId", migrationId);
-        return caseDetails;
-    }
-
     @TestInstance(TestInstance.Lifecycle.PER_CLASS)
     @Nested
-    class Fpla3262 {
-
-        private final String migrationId = "FPLA-3132";
+    class Dfpl164 {
+        private final String migrationId = "DFPL-164";
 
         @Test
         void shouldPerformMigration() {
             CaseData caseData = CaseData.builder()
-                .id(12345L)
+                .id(1626258358022834L)
                 .state(State.SUBMITTED)
-                .children1(wrapElements(
-                    Child.builder().build(), Child.builder().build(), Child.builder().build(),
-                    Child.builder().build(), Child.builder().build(), Child.builder().build(),
-                    Child.builder().build(), Child.builder().build(), Child.builder().build(),
-                    Child.builder().build(), Child.builder().build(), Child.builder().build(),
-                    Child.builder().build(), Child.builder().build(), Child.builder().build()
-
-                ))
-                .localAuthorities(wrapElements(LocalAuthority.builder().name("Some LA").build()))
+                .otherCourtAdminDocuments(
+                    wrapElements(
+                        CourtAdminDocument.builder()
+                            .document(DOCUMENT_REFERENCE).documentTitle("court-document1").build(),
+                        CourtAdminDocument.builder()
+                            .document(DOCUMENT_REFERENCE)
+                            .document(DocumentReference.builder().filename("LA Certificate.pdf").build())
+                            .documentTitle("court-document1").build()))
                 .build();
 
             CaseData responseData = extractCaseData(postAboutToSubmitEvent(buildCaseDetails(caseData, migrationId)));
```

Comment:
if you want to test the result of
```
caseDetails.getData().putAll(documentListService.getDocumentView(getCaseData(caseDetails)));
```
then I would suggest assigning this to it's own variable
```java
AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(buildCaseDetails(caseData, migrationId))
```
after that is complete you can check the value of `documentViewHMCTS` or whichever field is populated in the map. They will be strings so you can just assert that the string does not contain the file name
```java
assertThat(response.get("documentViewHMCTS")).asInstanceOf(InstanceOfAssertFactories.STRING).doesNotContain("LA Certificate.pdf");
```

Reply:
Done changes as suggested